### PR TITLE
Code cleanup & fix typos

### PR DESCRIPTION
**Pull request by pacion**
Small efficiency boost by replacing "\n" into '\n',
change variable name to make it easier to read and understand,
added extra spaces to make it easier to read.

### CODE REVIEW

File: installers/local/carbon.cpp [hunk 1]
```diff
@@ -1,7 +1,3 @@
-// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
-// Exceptions. See /LICENSE for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
```
Comment (junlarsen) [inline installers/local/carbon.cpp]: It looks like you accidentally removed the copyright header here

File: installers/local/carbon.cpp [hunk 2]
```diff
@@ -12,16 +8,19 @@ namespace path = llvm::sys::path;
...
     llvm::StringRef install_path = path::parent_path(exe);
     llvm::SmallString<256> prelude_file(install_path);
     path::append(prelude_file, "data", "prelude.carbon");
+    
     return Carbon::ExplorerMain(prelude_file, argc, argv);
   } else {
-    fprintf(stderr, "Unrecognized Carbon binary requested: %s", argv[0]);
-    return 1;
+    return fprintf(stderr, "Unrecognized Carbon binary requested: %s", argv[0]),
+           1;
```
Comment (junlarsen) [inline installers/local/carbon.cpp:28]: I'm not sure use of comma operator expression makes things easier to understand here :thinking:

**Review by junlarsen — COMMENTED** [review]
Most of the formatting is automatically done by Clang Format (and you'll see the pre-commit failing because of this)

You can set up the formatting to be done automatically before commit with pre-commit (see https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md#pre-commit)

File: installers/local/carbon.cpp (hunk 2 above)
Comment (pacion) [reply installers/local/carbon.cpp:24]: Would you explain me why?

File: installers/local/carbon.cpp (hunk 1 above)
Comment (pacion) [reply installers/local/carbon.cpp]: Made that a little bit messy, thanks for the link! I'll read it and then start to commit :))

**Pull request closed without merging**